### PR TITLE
Fix Max Upload Size handling

### PR DESCRIPTION
**Pull request by marcominerva**
## Motivation and Context (Why the change? What's the scenario?)
See https://github.com/microsoft/kernel-memory/pull/863#issuecomment-2443342730 and the comments below

### CODE REVIEW

File: service/Core/Configuration/ServiceConfig.cs [hunk 1]
```diff
@@ -30,18 +29,8 @@ public class ServiceConfig
     public Dictionary<string, HandlerConfig> Handlers { get; set; } = new();
 
     /// <summary>
-    /// The maximum allowed size in megabytes for a request body posted to the upload endpoint.
+    /// The maximum allowed size in bytes for a request body posted to the upload endpoint.
```
Comment (dluc) [inline service/Core/Configuration/ServiceConfig.cs:32]: as per conversation, I'd prefer sticking to "megabytes"

File: service/Core/Configuration/ServiceConfig.cs [hunk 2]
```diff
@@ -30,18 +29,8 @@ public class ServiceConfig
...
     /// If not set the solution defaults to 30,000,000 bytes (~28.6 MB) (ASP.NET default).
     /// </summary>
-    public long? MaxUploadSizeMb { get; set; } = null;
-
-    public long? GetMaxUploadSizeInBytes()
-    {
-        if (this.MaxUploadSizeMb.HasValue)
-        {
-            return Math.Min(10, this.MaxUploadSizeMb.Value) * 1024 * 1024;
```
Comment (dluc) [inline service/Core/Configuration/ServiceConfig.cs:42]: I think this is the only line that needs fixing, using `Math.Max`. I'd keep also the min of 10Mb in place.

**Review by dluc — CHANGES_REQUESTED** [review]
see comments

File: service/Core/Configuration/ServiceConfig.cs (hunk 2 above)
Comment (marcominerva) [reply service/Core/Configuration/ServiceConfig.cs:42]: So don't you want to handle scenarios where, for example, I need to limit the upload to 1 MB?

**Comment by marcominerva** [comment]
I think that this PR has been superseded by https://github.com/microsoft/kernel-memory/pull/896, so it can be closed.

**Pull request closed without merging**